### PR TITLE
Refactoring swift util test

### PR DESCRIPTION
**Pull request by gzhao9**
### Description

This PR Fixed the issue #8088

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: utils/src/test/java/com/cloud/utils/SwiftUtilTest.java [hunk 1]
```diff
@@ -215,15 +174,20 @@ public void testListContainerCmdWithStoragePolicyButNotSupportedByOperation() {
...
-        given(cfg.getStoragePolicy()).willReturn(null);
+        SwiftClientCfg cfg = CreateMockSwiftClientCfg(null);
 
         String uploadCmd = SwiftUtil.getSwiftContainerCmd(cfg, "swift", "list", "T-1");
         String expected = "/usr/bin/python swift -A swift.endpoint -U cs:sec-storage -K mypassword list T-1";
         assertThat(uploadCmd, is(equalTo(expected)));
     }
+
+    private SwiftClientCfg CreateMockSwiftClientCfg(String policy){
+        SwiftClientCfg cfg = mock(SwiftClientCfg.class);//
```
Comment (DaanHoogland) [inline utils/src/test/java/com/cloud/utils/SwiftUtilTest.java:185]:
```suggestion
        SwiftClientCfg cfg = mock(SwiftClientCfg.class);
```

**Review by DaanHoogland — APPROVED** [review]

**Review by rohityadavcloud — APPROVED** [review]
LGTM - need to check pkging build

**Comment by harikrishna-patnala** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Review by harikrishna-patnala — APPROVED** [review]
Code LGTM

**Pull request merged**